UI/ClientUI.cpp

```diff
@@ -476,101 +476,100 @@ namespace {
 
     // command-line options
     void AddOptions(OptionsDB& db) {
-        db.Add("show-fps",              UserStringNop("OPTIONS_DB_SHOW_FPS"),              false);
-        db.Add("limit-fps",             UserStringNop("OPTIONS_DB_LIMIT_FPS"),             true);
-        db.Add("max-fps",               UserStringNop("OPTIONS_DB_MAX_FPS"),               60.0,
-               RangedStepValidator<double>(1.0, 1.0, 240.0));
-        db.Add("limit-fps-no-focus",    UserStringNop("OPTIONS_DB_LIMIT_FPS_NO_FOCUS"),    true);
-        db.Add("max-fps-no_focus",      UserStringNop("OPTIONS_DB_MAX_FPS_NO_FOCUS"),      15.0,
+        db.Add("video.fps.shown",       UserStringNop("OPTIONS_DB_SHOW_FPS"), false);
+        db.Add("video.fps.max.enabled", UserStringNop("OPTIONS_DB_LIMIT_FPS"),             true);
+        db.Add("video.fps.max",         UserStringNop("OPTIONS_DB_MAX_FPS"),               60.0,
```

Comment:
might be better to call this `video.fps.limit` than `video.fps.max`, but not a major issue...

Reply:
I should address other uses of `limit` and `max`/`min` with unified terms. Currently favor `min` over something like `limit.lower`, but will look for alternatives.

Reply:
limits and maxes aren't necessarily synonymous ...